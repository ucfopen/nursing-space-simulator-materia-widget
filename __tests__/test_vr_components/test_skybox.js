import React from "react";
import renderer from "react-test-renderer";
import { Entity } from "aframe-react";
import Aframe from "aframe";

import Skybox from "../../src/js/components/assets/skybox";

describe("Skybox tests", () => {
	it("renders skybox correctly", () => {
		const skyboxTree = renderer.create(<Skybox />).toJSON();

		expect(skyboxTree).toMatchSnapshot();
	});
});
