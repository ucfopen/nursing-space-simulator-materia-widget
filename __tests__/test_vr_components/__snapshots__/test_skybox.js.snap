// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skybox tests renders skybox correctly 1`] = `<a-sky />`;
